test/e2e/specs/plans/plans__existing-free-site-upgrade.ts

```diff
@@ -19,7 +19,11 @@ import { apiDeleteSite } from '../shared';
 
 declare const browser: Browser;
 
-describe(
+/**
```

Comment:
**suggestion**: @claudiucelfilip  another way to disable a test without having to disable waking rules are to simply remove the "@group" from the JSDoc header.

Reply:
> **suggestion**: @claudiucelfilip another way to disable a test without having to disable waking rules are to simply remove the "@group" from the JSDoc header.

Thanks for this!